src/Microsoft.AspNetCore.Localization.Routing/Microsoft.AspNetCore.Localization.Routing.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why you're creating a new project for Routing? It will be nice to follow the structure that's already used in the repo. I think adding `RouteDataRequestCultureProvider.cs` is appropriate choice